R/crs.R

```diff
@@ -0,0 +1,98 @@
+#' Return the scale factor for a specific parallel in a Lambert Conic Conformal
+#' projection
+#'
+#' The scale factor is the distance distortion.
+#' In this case (a conic conformal projection) it holds in any direction.
+#'
+#' The function applies to normal LCC projections, not the oblique ones.
+#' The applied formulas are for the sphere, hence approximate.
+#'
+#' @return
+#' Numeric vector, same length as \code{par_deg}.
+#'
+#' @family functions to explore properties of coordinate reference systems
+#'
+#' @references
+#' Snyder J.P. (1987). Map Projections - A Working Manual.
+#' U.S. Geological Survey Professional Paper, Nr. 1395. United States
+#' Government Printing Office, Washington, 397 p.
+#'
+#' @param par_deg Numeric vector.
+#' The latitude in decimal degrees of one or more parallels for which to do the
+#' calculation.
+#' @param par1_deg Latitude of the first standard parallel of the LCC projection
+#' in decimal degrees.
+#' @param par2_deg Latitude of the second standard parallel of the LCC
+#' projection in decimal degrees.
+#'
+#' @examples
+#' sf::st_crs(31370)
+#' x <- seq(49.5, 52.5, 0.1)
+#' y <- scalefactor_lcc(x, 51.1666672333333, 49.8333339)
+#' data.frame(latitude = x, scalefactor = y)
+#'
+#' @author
+#' Floris Vanderhaeghe, \url{https://github.com/florisvdh}
```

Comment:
Note that you can add information below to the `authors` section of `_pkgdown.yml`

```
  Flors Vanderhaeghe:
    href: "https://github.com/florisvdh"
```

Reply:
It replaces `Floris Vanderhaeghe` with `[Floris Vanderhaeghe](https://github.com/florisvdh)` on the pkgdown website.

Reply:
see [2de1bc2](https://github.com/inbo/inbospatial/pull/1/commits/2de1bc2d03ee829dbc36e8e4f48ff136eee23598)